app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -51,20 +146,28 @@ def params_for_create
     # Verify Credentials
     #
     # args: {
+    #   "name" => String,
     #   "endpoints" => {
     #     "default" => {
-    #       "username" => String,
+    #       "default_userid" => String,
+    #       "default_hostname" => String,
+    #       "default_api_port" => Integer,
+    #       "default_security_protocol" => String,
+    #       "password" => String,
+    #     },
+    #     "amqp" => {
+    #       "amqp_hostname" => String,
+    #       "amqp_userid" => String,
+    #       "amqp_api_port" => String,
     #       "password" => String,
-    #       "server" => String,
-    #       "port" => Integer,
-    #       "protocol" => String
-    #     }
-    #   }
+    #     },
+    #   },
+    # }
     def verify_credentials(args)
       default_endpoint = args.dig("endpoints", "default")
 
       username, password, server, port, protocol = default_endpoint&.values_at(
-        "username", "password", "server", "port", "protocol"
+        "default_userid", "password", "default_hostname", "default_api_port", "default_security_protocol"
```

Comment:
This probably won't work anymore, but take a look at how I'm doing it in Amazon for inspiration

Reply:
Yeah I need to update this